### PR TITLE
Add fuzzy tests

### PR DESCRIPTION
**Pull request by jwirkus**
Required: https://github.com/Open-CAS/open-cas-linux/pull/1399

Signed-off-by: jwirkus <jakubx.wirkus@intel.com>

### CODE REVIEW

File: test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_script_add_core_device_try_add.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+from api.cas.cli import script_try_add_cmd, remove_detached_cmd
+from core.test_run import TestRun
+from test_tools.peach_fuzzer.peach_fuzzer import PeachFuzzer
+from tests.security.fuzzy.common.common import global_fuzz_iterations_count
+from tests.security.fuzzy.kernel.common.common import get_fuzz_config, run_cmd_and_validate
+
+iterations_count = global_fuzz_iterations_count if global_fuzz_iterations_count else 1000
+
+
+def test_fuzzy_add_core_device_try_add():
```
Comment (jwirkus) [inline test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_script_add_core_device_try_add.py:16]: Should this test be parametrized (eg. start and stop cache to reload kernel modules)?

File: test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_add_core_cache_id.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+@pytest.mark.parametrizex("cache_mode", CacheMode)
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+@pytest.mark.parametrizex("cleaning_policy", CleaningPolicy)
+@pytest.mark.parametrizex("unaligned_io", UnalignedIo)
+@pytest.mark.parametrizex("use_io_scheduler", UseIoScheduler)
+def test_fuzzy_add_core_cache_id(cache_mode, cache_line_size, cleaning_policy, unaligned_io,
+                                 use_io_scheduler):
+    """
+        title: Fuzzy test for casadm 'add core' command – cache id
+        description: Using Peach Fuzzer check Intel CAS ability of handling wrong cache id in
```
Comment (mmichal10) [inline test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_add_core_cache_id.py:33]: Please change all occurrences of "Intel CAS" to "Open CAS"

File: test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_add_core_core_id.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            output = run_cmd_and_validate(cmd, "Core id",
+                                          __is_value_valid(cmd.param.decode('ascii', 'ignore')))
+            if output.exit_code == 0:
+                with TestRun.step("Remove core"):
+                    core = Core(core_disk.path, cache.cache_id)
+                    TestRun.executor.run_expect_success(remove_core_cmd(str(cache.cache_id),
+                                                                        core.core_id))
+
+
+def __is_value_valid(parameter):
```
Comment (mmichal10) [inline test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_add_core_core_id.py:63]: Please rename the function to `__is_valid_core_id`

File: test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_add_core_core_id.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    try:
+        max_value = (1 << 12) - 1
+        if 0 <= int(parameter) <= max_value:
+            return True
+    except ValueError:
+        pass
+
+    return False
```
Comment (mmichal10) [inline test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_add_core_core_id.py:71]: How about 
```
min_core_id = 0
max_core_id = 4095
return parameter.isdigit() and 0 <= int(parameter) <= max_value

File: test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_script_add_core_device_try_add.py (hunk 1 above)
Comment (mmichal10) [reply test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_script_add_core_device_try_add.py:16]: Could you clarify what parameters you would like to add to the test?

Comment (jwirkus) [reply test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_script_add_core_device_try_add.py:16]: unaligned_io and use_io_scheduler in similar fashion as in test_fuzzy_start_cache_device line 41 (I guess other parameters don't matter in this case)

**Pull request closed without merging**